patches/001_email-verified.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/lib/profileMapper.js b/lib/profileMapper.js
+index 31ef24a..b503e97 100644
+--- a/lib/profileMapper.js
++++ b/lib/profileMapper.js
+@@ -1,4 +1,13 @@
+ module.exports = function (raw_data) {
++  /* Mozilla LDAP specific aliases handling */
++  var all_emails = [];
++  if (raw_data.mail) {
++    all_emails = all_emails.concat(raw_data.mail);
++  }
++  if (raw_data['zimbraAlias']) {
++    all_emails = all_emails.concat(raw_data['zimbraAlias']);
++  }
++
```

Comment:
I would probably make this entire thing:

```javascript
const all_emails = [].concat(raw_data.mail || [],
                             raw_data.zimbraAlias || [])
```

Reply:
Right, I figured I'd just capture the existing code that was in the forked repo in this patch file for the moment and worry about improving the code after we'd cutover to this patch based deployment method. 

I captured this suggestion in #7